### DISCUSSION TITLE
post/aliyun-oss-brushed-1000rmb-fees

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#8] [General]
# post/aliyun-oss-brushed-1000rmb-fees

大致经过 垂死病中惊坐起😱   事情发生在 8 月 8 日凌晨，凌晨三点我突然看见手机上的消息  我一开始是疑惑的，我的 OSS 是用来当做图床的，一个月也用不了几个钱 账号里记得还有 20 多块钱，怎么会这么快用完 然后我进阿里云一看，哇，我被人刷了？    最后发现被刷了 3.57 TB，请求

https://nickxu.me/post/aliyun-oss-brushed-1000rmb-fees

**Comment by yiliang114** [comment]
不知道为啥，我一张图片都看不见... 所以在国内这样托管图片可能还是会有网络问题？

**Reply by ZhangStudyLife** [reply]
裸连的话，打开隐私模式重新加载或者Ctrl+F5，我这里可以加载出来所有的文字，标签页左上角一直转圈圈，图片是加载不出来的；开了代理是可以秒加载的

**Reply by nickxudotme** [reply]
国内的确体验不好，但考虑大部分搞技术的都是 24h🪜，也没啥了
我同学的 blog 甚至直接 block 了所有的中国大陆访问

**Reply by ZhangStudyLife** [reply]
哈哈哈,我就写了几篇简单的hdu新生怎么注册三个平台的文章，因为群里一堆人在问各种奇葩问题，我就写了一篇（原以为一劳永逸），然后一堆人点都点不开，还被微信标记为色情网页，哈哈哈哈哈哈哈哈哈哈哈哈，太好笑了(不太能理解为什么很多人注册账号都有问题

**Comment by haoyizebo** [comment]
R2 的问题就是国内访问有时候会遇到问题，itdog 测速可能会是全红

**Comment by changhr2013** [comment]
我感觉你可以考虑一下 webp 格式，相对于 png，还能再省个一半的存储空间。

**Reply by nickxudotme** [reply]
合理的，但是我是在 Typora 粘贴图片 PicGo 就自动上传了，应该有插件能转换一下

**Comment by Arron196** [comment]
实际上r2同样有坑，因为404和带参数的访问也会消耗你的B类次数
